packages/ember-debug-global/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "composite": true,
     "rootDir": "src",
-    "outDir": "dist"
+    "outDir": "dist",
+    "tsBuildInfoFile": "tmp/.tsbuildinfo"
```

Comment:
ts-loader does not recognize this by default? I guess I assumed it would.

Reply:
It puts it at the root of each project by default (so `packages/ember-debug/.tsbuildinfo`) which is a bit annoying/distracting, so I used this setting to relocate it into `tmp` (so `packages/ember-debug/tmp/.tsbuildinfo`) so you can more easily ignore it in your editor's file tree.